packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -62,138 +95,208 @@ function WorklistDataGrid(props) {
     ariaLabel,
     columns,
     rows,
+    onCellSelect,
+    onRowSelect,
+    onClearSelectedRows,
+    onClearRowSelectionMode,
+    hasSelectableRows,
+    intl,
   } = props;
 
+  const grid = useRef();
   const focusedRow = useRef(0);
   const focusedCol = useRef(0);
-  const grid = useRef();
+  const rowSelectionMode = useRef(false); // used to detect change in row selection Mode.
+  const [currentSelectedCell, setCurrentSelectedCell] = useState(null);
+  const [isNavigationEnabled, setIsNavigationEnabled] = useState(true);
+  const displayedColumns = (hasSelectableRows ? [WorklistDataGridUtils.ROW_SELECTION_COLUMN] : []).concat(columns);
+
+  const theme = useContext(ThemeContext);
+
+  const isRowSelectionCell = (columnIndex) => (
```

Comment:
If we have a RowSelectionCell with an onRowSelect method, it would seem like this function would be unnecessary.